capnp/src/any_pointer_list.rs

```diff
@@ -87,7 +87,7 @@ impl <'a> Builder<'a> {
         Reader { reader: self.builder.into_reader() }
     }
 
-    pub fn get(self, index : u32) -> crate::any_pointer::Builder<'a> {
+    pub fn get(&self, index : u32) -> crate::any_pointer::Builder<'a> {
```

Comment:
Needs to be `&mut self` to prevent aliasing of mutable references.
(Similar comment applies elsewhere in this diff.)